docker-compose.yml

```diff
@@ -141,20 +141,24 @@ services:
       - ./test:/app/test
       - ./features:/app/features
 
-  react-native-cli:
-    build:
-      context: .
-      dockerfile: dockerfiles/Dockerfile.react-native-cli
+  react-native-cli-tool-maze-runner:
+    # TODO Use Maze Runner release
+    image: 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner:integration-interactive_cli-cli
     environment:
       DEBUG:
+      VERBOSE:
```

Comment:
Is `VERBOSE` used by anything? It looks the same as running without it to me

```suggestion
```

Reply:
VERBOSE and DEBUG are synonymous in Maze and I'm just following the established pattern here.  We should probably just take one of them when we next make a breaking change to reduce the amount of config kicking around.

Reply:
I'm not sure that's true; running with `VERBOSE` doesn't seem to do anything

Reply:
Ah, I was referring to the core MazeRunner logger itself, which contains:
```
  def initialize
    if ENV['VERBOSE'] || ENV['DEBUG']
      super(STDOUT, level: Logger::DEBUG)
```

But in this context it'll be limited by the absence of `VERBOSE` from the secondary `docker-compose.yml` file - for the test fixtures.  TBH I only put it in for consistency with existing entries, so let's talk IRL and see if we can just get rid of it from everywhere.